pkgs/development/python-modules/qcengine/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage, lib, fetchPypi, cacert, numpy, pydantic, pint,
+  networkx, pytestrunner, pytestcov, pytest
+} :
+
+buildPythonPackage rec {
+  pname = "qcelemental";
+  version = "0.19.0";
+
+  checkInputs = [ pytestrunner pytestcov pytest ];
+  propagatedBuildInputs = [ numpy pydantic pint networkx cacert ];
```

Comment:
What is `cacert` need for here?

Reply:
Older versions required it, as they downloaded molecular structures and constants from the web. The current versions apparently does not require it anymore. I've removed it.